buildenv/jenkins/omrMirror

```diff
@@ -24,9 +24,10 @@ pipeline {
     agent {label 'master'}
 
     environment {
-        HTTP        = 'https://'
-        SRC_REPO    = 'github.com/eclipse/omr.git'
-        TARGET_REPO = 'github.com/eclipse/openj9-omr.git'
+        HTTP         = 'https://'
+        SRC_REPO     = 'github.com/eclipse/omr.git'
+        TARGET_REPO  = 'github.com/eclipse/openj9-omr.git'
```

Comment:
This code change bothered me earlier as I couldn't figure out why it was showing up as a change.... and the answer is whitespace.  This is one of the big reasons why I'm strongly opposed to this kind of formatting.

Rather than this appearing as a single line of change - easy to review, easy to tell why it changed - it now looks like all 4 lines were modified.

I'm not going to say it has to be changed, but will ask that this kind of alignment is avoided in future commits